pkg/action/failover.go

```diff
@@ -311,6 +311,16 @@ func (ac *ActionController) activateClusterDuringDR(action *storkv1.Action, name
 		return
 	}
 
+	var failureStatus storkv1.ActionStatusType
```

Comment:
nit: assign the default values in else part. 

then overwrite only with `if rollback`.

Reply:
ohh makes sense. will address